vueui/src/components/header/modular-computer.vue

```diff
@@ -23,6 +23,11 @@
 export default {
   data() {
     return this.$root.$data
+  },
+  mounted: function() {
+    this.$nextTick(function() {
+      document.getElementById('content').style.height = 'calc(100vh - 72px)';
+    })
```

Comment:
Same situation as in the other files - this function should be unnecessary if you get the title bar height dynamically.